src/Files.App/Commands/Manager/CommandManager.cs

```diff
@@ -134,12 +190,14 @@ private void Action_PropertyChanging(object? _, PropertyChangingEventArgs e)
 						break;
 				}
 			}
-			private void Action_PropertyChanged(object? sender, PropertyChangedEventArgs e)
```

Comment:
Please don't discard `sender` parameters 🙂

Reply:
It's done.